src/components/common/helpers/workflowURL.ts

```diff
@@ -1,37 +1,50 @@
-
-export function getCDPipelineURL(appId: string, workflowId: string, ciPipelineId: string, isWebhookParent: boolean, cdPipelineId: string = null) {
+export function getCDPipelineURL(
+    appId: string,
+    workflowId: string,
+    ciPipelineId: string,
+    isWebhookParent: boolean,
+    cdPipelineId: string = null,
+) {
     if (cdPipelineId)
-        return `${workflowId}/${isWebhookParent? 'webhook':'ci-pipeline'}/${ciPipelineId}/cd-pipeline/${cdPipelineId}`;
-    else
-        return `${workflowId}/${isWebhookParent? 'webhook':'ci-pipeline'}/${ciPipelineId}/cd-pipeline`;
+        return `${workflowId}/${
+            isWebhookParent ? 'webhook' : 'ci-pipeline'
+        }/${ciPipelineId}/cd-pipeline/${cdPipelineId}`
+    else return `${workflowId}/${isWebhookParent ? 'webhook' : 'ci-pipeline'}/${ciPipelineId}/cd-pipeline`
 }
 
-export function getCIPipelineURL(appId: string, workflowId: string, ciPipelineId: string | number = null) {
-    if (ciPipelineId)
-        return `${workflowId}/ci-pipeline/${ciPipelineId}`;
-    else
-        return `${workflowId}/ci-pipeline`;
+export function getPrefixURL(appId: string) {
+    return `/app/${appId}/edit/workflow/`
 }
 
-export function getExCIPipelineURL(appId: string, workflowId: string, ciPipelineId: string = null) {
-    if (ciPipelineId)
-        return `${workflowId}/external-ci/${ciPipelineId}`;
-    else
-        return `${workflowId}/external-ci`;
+export function getCIPipelineURL(
+    appId: string,
+    workflowId: string,
+    ciPipelineId: string | number = null,
+    isGitNotConfigured: boolean,
+) {
+    let prefixURL = ''
+    if (isGitNotConfigured) {
+        prefixURL = getPrefixURL(appId)
```

Comment:
No need of additional function replace this directly with``/app/${appId}/edit/workflow/``

Reply:
Also, we can write it something like 
```
const prefixURL = isGitNotConfigured ? `/app/${appId}/edit/workflow/` : ''
    return `${prefixURL}${workflowId}/ci-pipeline${ciPipelineId ? `/${ciPipelineId}` : ''}`
```

Reply:
Can be done in multiple functions in this file?

Reply:
done at all occurrences